### PR TITLE
Change addon name locator

### PR DESCRIPTION
**Pull request by jrbenny35**
This should fix the fails on nightly.

### CODE REVIEW

File: foxpuppet/windows/browser/notifications/addons.py [hunk 1]
```diff
@@ -33,8 +33,8 @@ def addon_name(self):
         """
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
             if self.window.firefox_version >= 55:
-                return self.root.find_anonymous_element_by_attribute(
-                    'anonid', 'description').find_element(
+                return self.root.find_element(
+                    By.ID, 'addon-webext-perm-header').find_element(
```
Comment (davehunt) [inline foxpuppet/windows/browser/notifications/addons.py]: If we're not using anonymous nodes, then locators should be defined as class variables. As we're effectively replacing `_addon_name_locator` I think we can just replace this class variable within this if block.

File: foxpuppet/windows/browser/notifications/addons.py [hunk 2]
```diff
@@ -33,9 +35,9 @@ def addon_name(self):
         """
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
             if self.window.firefox_version >= 55:
-                return self.root.find_anonymous_element_by_attribute(
-                    'anonid', 'description').find_element(
-                        By.CSS_SELECTOR, '.addon-webext-name').text
+                return self.root.find_element(
+                    *self._web_ext_header_locator).find_element(
+                        *self._web_ext_name_locator).text
```
Comment (davehunt) [inline foxpuppet/windows/browser/notifications/addons.py]: Are you not able to reference this element directly using `#addon-webext-perm-header .addon-webext-name`? The only reason an intermediary node was used before was due to the parent node being anonymous.

File: foxpuppet/windows/browser/notifications/addons.py [hunk 3]
```diff
@@ -22,6 +22,8 @@ class AddOnInstallConfirmation(BaseNotification):
 
     _addon_name_locator = (
         By.CSS_SELECTOR, '#addon-install-confirmation-content label')
+    _web_ext_header_locator = (By.ID, 'addon-webext-perm-header')
+    _web_ext_name_locator = (By.CSS_SELECTOR, '.addon-webext-name')
```
Comment (davehunt) [inline foxpuppet/windows/browser/notifications/addons.py]: I think we should just reassign the value of `_addon_name_locator` within the conditional logic. Once all supported Firefox versions use the same locator we just remove the conditional block and revert to a single value for the locator.

**Comment by jrbenny35** [comment]
I need #95 merged before this test can work properly.

File: foxpuppet/windows/browser/notifications/addons.py [hunk 4]
```diff
@@ -33,11 +31,15 @@ def addon_name(self):
         """
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
             if self.window.firefox_version >= 55:
-                return self.root.find_anonymous_element_by_attribute(
-                    'anonid', 'description').find_element(
-                        By.CSS_SELECTOR, '.addon-webext-name').text
+                _addon_name_locator = (
+                    By.CSS_SELECTOR, '#addon-webext-perm-header > \
+                                      .addon-webext-name')
```
Comment (davehunt) [inline foxpuppet/windows/browser/notifications/addons.py]: One of these locators should still be on line 22. I would suggest this one, as it should ultimately become the only locator once the changes ride the trains to release and eventually ESR.

File: foxpuppet/windows/browser/notifications/addons.py [hunk 5]
```diff
@@ -33,11 +31,15 @@ def addon_name(self):
...
+                return self.root.find_element(*_addon_name_locator).text
             else:
-                label = self.root.find_element(*self._addon_name_locator)
+                _addon_name_locator = (
+                    By.CSS_SELECTOR, '#addon-install-confirmation-content \
+                                      label')
```
Comment (davehunt) [inline foxpuppet/windows/browser/notifications/addons.py]: Can we avoid splitting the string over multiple lines? If this doesn't fit in 80 characters, split between arguments such as:

```python
_addon_name_locator = (
    By.CSS_SELECTOR,
    '#addon-install-confirmation-content label')
```

**Comment by davehunt** [comment]
Are the Travis CI failures due to [bug 1376586](https://bugzilla.mozilla.org/show_bug.cgi?id=1376586) or something else?

**Comment by jrbenny35** [comment]
> Are the Travis CI failures due to bug 1376586 or something else?

Yes

**Pull request merged**